packages/expo-video/src/VideoView.types.ts

```diff
@@ -127,3 +127,43 @@ export interface VideoViewProps extends ViewProps {
    */
   startsPictureInPictureAutomatically?: boolean;
 }
+
+/**
+ * Specifies which type of DRM to use. Android supports Widevine, PlayReady and ClearKey.
```

Comment:
```suggestion
 * Specifies which type of DRM to use. Android supports Widevine, Playready and ClearKey.
```

But it might be better to say what is not supported on Android, i.e. "FairPlay is not supported on Android"

Reply:
I forgot to update the iOS part of the comment, iOS supports only FairPlay. So now it looks like this `Specifies which type of DRM to use. Android supports Widevine, PlayReady and ClearKey, iOS supports FairPlay.` 

I think the correct spelling is [`PlayReady` ](https://en.wikipedia.org/wiki/PlayReady)